### DISCUSSION TITLE
Gradle build error: unmappable character encoding

### DISCUSSION THREAD

**Discussion by Apoorvyash** [oppia/oppia-android#5045] [Q&A (Installation)]
### Operating System

Windows 11

### Android Studio Version

Flamingo | 2022.2.1

### RAM size

8

### Free disk space

15 GB

### Which step are you stuck on?

Hi,
 I have completed the setup following all steps for oppia android, and also ran git bash/setup.sh , android api level is 31 and 34.0.0 rc1 is installed, when I ran the gradle build , it failed with multiple error saying unmappable character for encoding Cp1252 and more.

### Error Log

```bash
error: unmappable character for encoding Cp1252
   * Corresponds to the Hindi (हिन�?दी) macro language. IETF BCP 47 language tag: hi.
                                          ^
```

### Approaches already used to resolve the issue

I tried to find the solution on stack overflow, tried to change the encoding format, and restarted build process after invalidating caches

**Comment by MohitGupta121** [comment]
@Apoorvyash I request you to please follow this https://github.com/oppia/oppia-android/discussions/4984#discussioncomment-5930244

**Reply by Rd4dev** [reply]
Hi @MohitGupta121, 

I tested the solutions provided in [comment #4984](https://github.com/oppia/oppia-android/discussions/4984#discussioncomment-5957004). As you mentioned, the issue is connected to the jetifier, and following your suggestion, I attempted to compile using a JDK version lower than 15. Specifically, I used **JDK-11**, which is conveniently available for download directly from Android Studio. 

Additionally, I tested the same solution with Android Studio Dolphin and Android Studio Giraffe versions, and both of them built successfully when compiled with JDK-11. 

Therefore, it seems that **using JDK-11 is a suitable solution**. 

**Operating System**
Windows 11

**Android Studio Version**
Flamingo | 2022.2.1

Followed are the images attached with problem and solution references

Android Studio Version:
![1](https://github.com/oppia/oppia-android/assets/122200035/a0bf3c50-9465-4aa0-873d-f7c7a9f18b47)

Compiled with JDK-17
![2](https://github.com/oppia/oppia-android/assets/122200035/62cd3a75-08fb-42cf-81d3-18ea0246ffb1)

Compiled with JDK-11
![3](https://github.com/oppia/oppia-android/assets/122200035/5e17b518-94b6-4088-9ae8-f166a88716b3)

Thanks.

**Comment by Apoorvyash** [comment]
I'll try it and get back , thanks for reaching out.
